packages/components/format-date/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './src/format-date.js';
+export * from './src/format.js';
```

Comment:
Please export this separately, so that if somebody only wants to use the `format()` function, the web component isn't also loaded.